simulation/simulation_test.go

```diff
@@ -439,6 +453,12 @@ func TestAppImportExport(t *testing.T) {
 	}
 }
 
+// TestAppSimulationAfterImport tests the application simulation after importing the state exported from a previous simulation run.
+// It does the following steps
+// 1. It runs a full simulation and exports the state
+// 2. It creates a new app, and db
+// 3. It imports the exported state into the new app
+// 4. It runs a simulation on the new app and verifies that there is error in the second simulation
```

Comment:
"there is an error" or "there's NO error" ? 😅 
```suggestion
// 4. It runs a simulation on the new app and verifies that there is error in the second simulation
```